sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/datastore/DatastoreV1.java

```diff
@@ -345,25 +354,12 @@ static RunQueryRequest makeRequest(Query query, @Nullable String namespace) {
     }
 
     /**
-     * Note that only {@code namespace} is really {@code @Nullable}. The other parameters may be
-     * {@code null} as a matter of build order, but if they are {@code null} at instantiation time,
-     * an error will be thrown.
```

Comment:
build() is an implementation detail, sufficient validation happens in validate().